pkg/logging/logging.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2021 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package fybriklogging
+
+import (
+	"encoding/json"
+
+	"github.com/go-logr/logr"
+)
+
+// LogType indicates the types of log entries supported.  Logging packages typically provide separate function
+// calls for the different log types.
+type LogType string
+
+const (
+	ERROR LogType = "error" // Errors encountered in the code
+	INFO  LogType = "info"  // Info logs are things you want to tell the user or administrator which are not errors.
```

Comment:
We need more log types, e.g. "audit"

Reply:
I adopted the approach recommended by github.com/go-logr/logr, which is based on [Dave Cheney's approach](https://dave.cheney.net/2015/11/05/lets-talk-about-logging) as described in their readme.